learning-bitmark/contributing-to-bitmark/bitmark-software-development-process.md

```diff
@@ -0,0 +1,127 @@
+# Current development workflow
```

Comment:
Right now this file needs an intro which makes it clear who it's directed to, what they can get out of this, and what its contents are.